docs/en_US/awesome/tools.md

```diff
@@ -0,0 +1,45 @@
+# Awesome Tools
+
+## Online Tools
+
+> UUID Generator
+
+<Uuid />
+
+There's a third-part UUID generator [uuidgenerator.net](https://www.uuidgenerator.net)
+
+## GUI Client
+
+<Tool
+    url="https://github.com/2dust/v2rayNG"
+    name="V2RayNG"
+    :platforms="['android']"
+    description="V2RayNG is an Android APP based on V2Ray core, VPN connections can be created with VMess."
+/>
+
+<Tool
+    url="https://github.com/2dust/v2rayN"
+    name="V2rayN"
+    :platforms="['win']"
+    description="V2RayN is a V2Ray client running on Windows."
+/>
+
+<Tool
+    url="https://github.com/yanue/V2rayU"
+    name="V2rayU"
+    :platforms="['mac']"
+    description="V2rayU，a macOS client based on V2Ray core. It's programmed with Swift 4.2，supports for VMess、Shadowsocks、SOCKS5 protocols etc，subscription、QR code、importing from clipboard、custom config、QR sharing are also supported"
+/>
+
+<Tool
+    url="https://github.com/Qv2ray/Qv2ray"
+    name="Qv2ray"
+    :platforms="['linux','win','mac']"
+    description="Cross-platform client of V2Ray，supports for Linux、Windows、macOS，SSR / Trojan / Trojan-Go / NaiveProxy can be supported by plugins，no batching speed test，no auto update，not for beginners."
+/>
+
+## Others
+
+> V2Ray Rules Dat
+
+Enhanced version of V2Ray routing rules file，it could take place of official `geoip.dat` and `geosite.dat` [GitHub](https://github.com/Loyalsoldier/v2ray-rules-dat)

```

Comment:
```suggestion
Enhanced version of V2Ray routing rules file. It could take place of official `geoip.dat` and `geosite.dat` [GitHub](https://github.com/Loyalsoldier/v2ray-rules-dat)

```